audit-triplestore/src/test/java/org/fcrepo/camel/audit/triplestore/integration/AuditSparqlIT.java

```diff
@@ -0,0 +1,283 @@
+                USER,
+                USER_AGENT
+            );
+
+        template.sendBody("direct:clear", null);
+        template.sendBodyAndHeaders(null, getEventHeaders());
+
+        template.sendBodyAndHeader("direct:query", null, Exchange.HTTP_QUERY, "query=SELECT ?o WHERE { ?s <" + PREMIS + "hasEventRelatedAgent> ?o }");
+
+        sparqlQueryEndpoint.assertIsSatisfied();
+        sparqlUpdateEndpoint.assertIsSatisfied();
+    }
+
+    @Test
+    public void testAuditEventAllTriples() throws Exception {
+        sparqlUpdateEndpoint.expectedMessageCount(2);
+        sparqlQueryEndpoint.expectedHeaderReceived(Exchange.HTTP_RESPONSE_CODE, 200);
+
+        sparqlQueryEndpoint.expectedMessageCount(8);
+        sparqlQueryEndpoint.expectedHeaderReceived(Exchange.HTTP_RESPONSE_CODE, 200);
+
+        template.sendBody("direct:clear", null);
+        template.sendBodyAndHeaders(null, getEventHeaders());
+
+        template.sendBodyAndHeader("direct:query", null, Exchange.HTTP_QUERY, "query=SELECT ?o WHERE { ?s ?p ?o }");
+
+        sparqlQueryEndpoint.assertIsSatisfied();
+        sparqlUpdateEndpoint.assertIsSatisfied();
+    }
+
+    @Test
+    public void testAuditTypeTriples() throws Exception {
+        sparqlUpdateEndpoint.expectedMessageCount(2);
+        sparqlUpdateEndpoint.expectedHeaderReceived(Exchange.HTTP_RESPONSE_CODE, 200);
+
+        sparqlQueryEndpoint.expectedMessageCount(2);
+        sparqlQueryEndpoint.expectedHeaderReceived(Exchange.HTTP_RESPONSE_CODE, 200);
+        sparqlQueryEndpoint.expectedBodiesReceivedInAnyOrder(
+                "http://www.loc.gov/premis/rdf/v1#Event",
+                "http://fedora.info/definitions/v4/audit#InternalEvent",
+                "http://www.w3.org/ns/prov#InstantaneousEvent"
+            );
+
+        template.sendBody("direct:clear", null);
+        template.sendBodyAndHeaders(null, getEventHeaders());
+
+        template.sendBodyAndHeader("direct:query", null, Exchange.HTTP_QUERY, "query=SELECT ?o WHERE { ?s <" + RDF + "type> ?o }");
+
+        sparqlQueryEndpoint.assertIsSatisfied();
+        sparqlUpdateEndpoint.assertIsSatisfied();
+    }
+
+    @Override
+    protected RouteBuilder createRouteBuilder() throws Exception {
+
+        final Namespaces ns = new Namespaces("sparql", "http://www.w3.org/2005/sparql-results#");
+
+        final XPathBuilder xpath = new XPathBuilder(
+                "//sparql:result/sparql:binding[@name='o']");
+        xpath.namespaces(ns);
+
+        final XPathBuilder uriResult = new XPathBuilder(
+                "/sparql:binding/sparql:uri");
+        uriResult.namespaces(ns);
+
+        final XPathBuilder literalResult = new XPathBuilder(
+                "/sparql:binding/sparql:literal");
+        literalResult.namespaces(ns);
+
+        return new RouteBuilder() {
+            public void configure() throws IOException {
+                final String fuseki_url = "localhost:" + System.getProperty("fuseki.dynamic.test.port", "3030");
```

Comment:
`System.getProperty("fuseki.dynamic.test.port", "3030")` is the same as the `FUSEKI_PORT` defined at the top of this class, no?